### PR TITLE
[docker] Restore mtime of .git* files

### PR DESCRIPTION
**Pull request by WonyoungChoi**
Restore mtime of .git* files to OLD TIME together because some ninja build uses the `.git/logs/HEAD` files to get the version information. This affects the incremental build.

### CODE REVIEW

File: ci/docker/tizen/tools/cache-checksum.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [[ "$COMMAND" == "restore" ]]; then
     fi
 
     # Set mtime of files in $TARGET_DIRS to OLD time.
-    for d in $(find $TARGET_DIRS -type d -not -path "*/.git*"); do
+    for d in $(find $TARGET_DIRS -type d); do
```
Comment (HakkyuKim) [inline ci/docker/tizen/tools/cache-checksum.sh:43]: If mtime is temporary set to OLD time for all .git files, then shouldn't we also do validation with checksum?

File: ci/docker/tizen/tools/cache-checksum.sh [hunk 2]
```diff
@@ -40,7 +40,7 @@ if [[ "$COMMAND" == "restore" ]]; then
...
         touch -c -m -d @1600000000 $d/* &
```
Comment (HakkyuKim) [inline ci/docker/tizen/tools/cache-checksum.sh:44]: I'm just curious, is there a specific meaning to the @1600000000 time or is it just a random time in the past?

Comment (WonyoungChoi) [reply ci/docker/tizen/tools/cache-checksum.sh:44]: > I'm just curious, is there a specific meaning to the @1600000000 time or is it just a random time in the past?

This is an arbitrary time value. In fact, any value before the time the cache was created is fine.

File: ci/docker/tizen/tools/cache-checksum.sh (hunk 1 above)
Comment (WonyoungChoi) [reply ci/docker/tizen/tools/cache-checksum.sh:43]: From the perspective of the Incremental build, there is no need to forward the .git files to ninja's changed files input.

**Review by swift-kim — APPROVED** [review]

**Pull request merged**